README.md

```diff
@@ -6,6 +6,20 @@ This framework encapsulates everything required to display the Authentication UI
 
 Plus: WordPress.com *signup*  is supported.
 
+## Setup
+
+```sh
+# Install required ruby 2.6.5 
+rbenv install 2.6.5
```

Comment:
I'd suggest `rbenv install -s`

By not specifying the version, it's read from the `.ruby-version` file, so this instruction won't need to be updated if we update the project's ruby version.

The `-s` flag says "don't do anything if that version is already installed", which can be handy for folks who just copy/paste commands 😁

Reply:
Oooh nice! Great suggestion, thank you!